packages/strapi-plugin-upload/controllers/upload/api.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
   async find(ctx) {
     const method = _.has(ctx.query, '_q') ? 'search' : 'fetchAll';
 
-    const files = await strapi.plugins.upload.services.upload[method](ctx.query);
+    const files = await strapi.plugins.upload.services.upload[method](ctx.query, []);
```

Comment:
here you shouldn't add it as it would ba a breaking change. The goal is that users add autoPopulate false if they don't want the data in their api but not the other way arround :)

Reply:
so the goal is just add it only for admin api?

Reply:
as far as I can see, both controllers use the same service. if we add `populate` parameter for the service then we should provide value for it. if adding an empty array for `populate` param from api controller is a breaking change then I can suggest to add a default param value for the service itself

Reply:
if the populate arg is undefined it will keep the same behavior as before so no need to provide it. it is optional in the query

Reply:
thanks for explanation, reverted back)